apps/i18n/common/en_us.json

```diff
@@ -1874,7 +1874,7 @@
   "reset": "Reset",
   "resetProgram": "Reset",
   "resetPassword": "Reset password",
-  "resetTeacherPasswordTooltip": "You cannot reset passwords for teacher accounts in your section.",
```

Comment:
This string is no longer needed, as the new changes make this message necessary,

Reply:
I'm wrong, missed another consumer of this key during a global project search. Will be adding it back in.

Reply:
I'm actually curious what other consumer of this key exists? If it is the `PasswordReset` component, that will never render this key now either due to your changes, yes?

Reply:
It's used for Password Reset https://github.com/code-dot-org/code-dot-org/blob/nicklathe/P20-22-pic-login/apps/src/templates/manageStudents/PasswordReset.jsx#L127

Reply:
^^ Specifically with email login

Reply:
Oh, email password resets also use this and show that tooltip? I see.